blocks/hooks/custom-class-name.js

```diff
@@ -27,7 +27,8 @@ import InspectorControls from '../inspector-controls';
  * @return {Object} Filtered block settings.
  */
 export function addAttribute( settings ) {
-	if ( hasBlockSupport( settings, 'customClassName', true ) ) {
+	if ( hasBlockSupport( settings, 'className', true ) &&
```

Comment:
I think `className` and `customClassName` are two distinct things and should not be both checked in this hook:

 - className is for the automatically generateClassname
 - customClassName is to allow people to add classNames via the inspector control.

I think we have some blocks with className: false where we still want the custom className (heading, list, paragraph)

Reply:
You are totally right, thank you for pointing that out. I updated this PR to just disable customClassName on the freeform block.